modules/claims_api/app/sidekiq/claims_api/v2/poa_form_builder_job.rb

```diff
@@ -29,7 +29,7 @@ def perform(power_of_attorney_id, form_number, rep_id)
           upload_to_vbms(power_of_attorney, output_path)
         end
 
-        ClaimsApi::PoaUpdater.perform_async(power_of_attorney.id)
+        ClaimsApi::PoaUpdater.perform_async(power_of_attorney.id, icn_for_vanotify, rep)
```

Comment:
https://github.com/department-of-veterans-affairs/vets-api/pull/18687/files#r1781912411 you get the idea.